spec/models/benefit_group_spec.rb

```diff
@@ -613,6 +613,41 @@
   end
 
   describe BenefitGroup, dbclean: :after_each do
+    let(:benefit_group) { FactoryGirl.create(:benefit_group)}
```

Comment:
you're making some test scaffolding mistakes.

your benefit group should be associated to the plan via reference_plan